plugins/reach/lib/api/data/vendorCatalogItem/KalturaVendorDocumentEnrichmentCatalogItem.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * @package plugins.reach
+ * @subpackage api.objects
+ */
+class KalturaVendorDocumentEnrichmentCatalogItem extends KalturaVendorCatalogItem
+{
+	protected function getServiceFeature(): int
```

Comment:
lets add here also DocumentEnrichmentType enum that currently hold only MARKDOWN_CONVERSIOn